app/views/events/show.html.erb

```diff
@@ -5,8 +5,10 @@
   <dd><%= @event.event_type.name %></dd>
   <dt><strong>Created At:</strong></dt>
   <dd><%= date_and_time @event.created_at %></dd>
-  <dt><strong>Is done:</strong></dt>
+  <dt><strong>Is Done:</strong></dt>
```

Comment:
This should be sentence casing (as should other labels), so it was correct before as 'Is done:'